code/modules/religion/religion_structures.dm

```diff
@@ -67,7 +67,7 @@
 /obj/structure/altar_of_gods/proc/get_chaplains()
 	var/chaplain_string = ""
 	for(var/mob/living/carbon/human/potential_chap in GLOB.player_list)
-		if(potential_chap.key && potential_chap.mind?.assigned_role == "Chaplain")
+		if(potential_chap.key && is_chaplain_job(potential_chap.mind?.assigned_role))
```

Comment:
this could be turned into a high priest holy check too?

Reply:
and then you could remove the is_chaplain_job define

Reply:
That's fine, but that needs to be done on a different PR, or this one will be even less reviewable.